modules/administration-guide/partials/con_che-postgresql.adoc

```diff
@@ -1,21 +1,28 @@
 // Module included in the following assemblies:
 //
-// {prod-id-short}-workspace-controller
+// {prod-id-short}-workspace-controller-with-che-server
 
 [id="{prod-id-short}-postgresql_{context}"]
 = {prod-short} and PostgreSQL
 
-The PostgreSQL database is a prerequisite to configure {prod-short}. The {prod-short} administrator can choose to connect {prod-short} to an existing PostgreSQL instance or let the {prod-short} deployment start a new dedicated PostgreSQL instance.
+The PostgreSQL database is a prerequisite for {prod-short} server and {identity-provider}.
 
-The {prod-short} server uses the database to persist user configurations (workspaces metadata, Git credentials). {identity-provider} uses the database as its back end to persist user information.
+The {prod-short} administrator can choose to connect {prod-short} to: 
 
-[cols=2*]
-|===
-ifeval::["{project-context}" == "che"]
-| Source code
-| link:{link-postgres-dockerfile-location}[{prod-short} Postgres]
-endif::[]
+* An existing PostgreSQL instance. 
+* Let the {prod-short} deployment start a new dedicated PostgreSQL instance.
+
+Services are consuming the database for following purposes:
```

Comment:
"consume" is a bit of a strong verb, maybe out of place here. It also might be considered anthropomorphism. 

Consider something like, `Services use the database for the following purposes`

Note, you were missing a `the` in front of `following purposes` :)